app/models/user.rb

```diff
@@ -4,7 +4,9 @@ class User < ApplicationRecord
   validates_presence_of :name
 
   def self.top_review_users
-    joins(:reviews).select("count(reviews.id) as count_reviews, users.*").group(:id).order("
-    count_reviews DESC").limit(3)
+    joins(:reviews).select("count(reviews.id) as count_reviews, users.*")
```

Comment:
sweet thanks for this. I know its not a big deal just think it looks a bit cleaner.